client/gui/CGuiHandler.cpp

```diff
@@ -284,6 +284,8 @@ void CGuiHandler::handleCurrentEvent()
 		switch(current->button.button)
 		{
 		case SDL_BUTTON_LEFT:
+		{
+			auto doubleClicked = false;

```

Comment:
I think that using auto for bool's is a bit unnecessary, but okay, can stay as it.